src/app/containers/Bulletin/index.jsx

```diff
@@ -58,30 +57,30 @@ BulletinImage.defaultProps = {
 const BulletinContainer = ({ item, lazyLoadImage }) => {
   const { script, service, dir, translations } = useContext(ServiceContext);
 
-  const headline = pathOr(null, ['name'], item);
-  const ctaLink = pathOr(null, ['uri'], item);
+  const headline = item?.name || null;
```

Comment:
Hope you don't mind me asking this (I know this is a draft PR, but I am just curious). 

If `item` is `null` or `undefined`, then `item?.name` = `undefined`. 

Is `null` preferable to `undefined` in this case (and all cases below)? Is it really required (just wondering if we can get away from needing to do the `...|| null` bit.

Thanks

Reply:
Yeah I wasn't too sure myself why these were all defaulting to `null`, but decided to keep that in these examples at least. It would otherwise default to `undefined` as you described.

Reply:
Probably best to just keep the functionality as it was before... or is it worth checking with the original author to see why the values were defaulted to `null`?

Out of interest, are you using a tool to help you with the conversion to optional chaining, or is it manual? Wondering if there is a way of automating this to prevent / reduce human errors. @andrewscfc mentioned a "code mod" which could be useful in this case?

Reply:
I've never used this but I'd suggest any change like this should be done in a mostly automated fashion: https://github.com/facebook/codemod